src/config.rs

```diff
@@ -133,7 +134,9 @@ pub fn parse_config() -> Result<Config, Error> {
         )?
         .0,
 
-        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 15021),
+        // admin API should only be accessible over localhost
+        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::LOCALHOST), 15000),
```

Comment:
since you already have a merge conflcit: can we make this on 0.0.0.0 until we split stats off? otherwise stats are broken until we do.

Reply:
Got it; will do